<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>487</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>487</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>487</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel#caption,#params {
	color: rgb(255, 85, 255);
	font: 16px Comic Sans MS;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineWeaponName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineWeaponDescription">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Название оружия</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Описание оружия на русском</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Принцип действия, что-то уточняющее</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPNGLoad">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PNG</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="zonePNG"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="zonePNGShoot"/>
   </widget>
   <widget class="QPushButton" name="pushButtonPNGShootLoad">
    <property name="geometry">
     <rect>
      <x>514</x>
      <y>160</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PNG выстрел</string>
    </property>
   </widget>
   <widget class="QLabel" name="params">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Параметры</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinDmgMin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Урон МИН</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Урон МАКС</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinDmgMax">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioCrits">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Критует</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="lineWeaponPrincipe">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Крит Урон МИН</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinCritDmgMin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Крит Урон МАКС</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinCritDmgMax">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="zonePNGProj"/>
   </widget>
   <widget class="QPushButton" name="pushButtonPNGProjectileLoad">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>250</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PNG прожектайл (есть?)</string>
    </property>
   </widget>
   <widget class="QLabel" name="caption">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Еще одно бесполезное оружие в игру </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineProjectileType">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Тип прожектайлов в игре</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Кулдаун на стрельбу (сек)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>390</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Время перезарядки (сек)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Кол-во прожектайлов в игре</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinProjectileAmount">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinClipSize">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>310</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>290</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Обойма в единицах</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>340</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Патронов за выстрел</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinAmmoPerShoot">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>360</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>390</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Кулдаун на след. урон (сек)</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>190</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="zonePNGAdd"/>
   </widget>
   <widget class="QPushButton" name="pushButtonPNGAdditionalLoad">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>250</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Еще какие-то PNG</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>280</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="zonePNGAdd2"/>
   </widget>
   <widget class="QPushButton" name="pushButtonPNGAdditionalLoad_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>340</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Еще какие-то PNG</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>390</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Макс. угол разброса (градусы)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinMaxScatter">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>410</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dspDmgCooldown">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>410</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dspShootCooldown">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>360</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dspReloadTime">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>410</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Действия</string>
    </property>
    <addaction name="weapNew"/>
    <addaction name="weapSave"/>
    <addaction name="weapSaveAs"/>
    <addaction name="weapLoadFromFile"/>
    <addaction name="weapExport"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="weapSave">
   <property name="text">
    <string>&amp;Сохранить оружие</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="weapExport">
   <property name="text">
    <string>&amp;Экспортировать в картинку</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="weapNew">
   <property name="text">
    <string>&amp;Новое оружие</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="weapLoadFromFile">
   <property name="text">
    <string>&amp;Загрузить оружие</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="weapSaveAs">
   <property name="text">
    <string>Сохранить &amp;как</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>lineWeaponName</tabstop>
  <tabstop>lineWeaponDescription</tabstop>
  <tabstop>lineWeaponPrincipe</tabstop>
  <tabstop>spinDmgMin</tabstop>
  <tabstop>spinDmgMax</tabstop>
  <tabstop>radioCrits</tabstop>
  <tabstop>spinCritDmgMin</tabstop>
  <tabstop>spinCritDmgMax</tabstop>
  <tabstop>lineProjectileType</tabstop>
  <tabstop>spinProjectileAmount</tabstop>
  <tabstop>spinClipSize</tabstop>
  <tabstop>spinAmmoPerShoot</tabstop>
  <tabstop>spinMaxScatter</tabstop>
  <tabstop>pushButtonPNGLoad</tabstop>
  <tabstop>pushButtonPNGShootLoad</tabstop>
  <tabstop>pushButtonPNGProjectileLoad</tabstop>
  <tabstop>pushButtonPNGAdditionalLoad</tabstop>
  <tabstop>pushButtonPNGAdditionalLoad_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
